### ISSUE TITLE
CentOS/Alma/Rocky : Login screen logo is too large after upgrade from xrdp v0.9.x to v0.10.x

### ISSUE THREAD

**Issue by 800118** [neutrinolabs/xrdp#3137]
### xrdp version

0.10.0-4

### Detailed xrdp version, build options

~~~
Paste the result between `~~~`.  Please DO NOT remove `~~~`!
~~~

### Operating system & version

CentOS 8.5.2111

### Installation method

dnf / apt / zypper / pkg / etc

### Which backend do you use?

xvnc

### What's your client?

windows  remote desktop

### Area(s) with issue?

Other

### Steps to reproduce

When logging into Centos8 using Windows remote desktop, the login interface displays an exception and the user box cannot be selected for input
![2024-06-24_191632](https://github.com/neutrinolabs/xrdp/assets/84504466/b46b8d85-aac5-4f20-b3bf-99f30eb3281b)

### ✔️ Expected Behavior

Like previous versions

### ❌ Actual Behavior

Display abnormalities

**Comment by matt335672** [comment]
@Aerithlon - can you fill in all the missing information above? There's not enough to go on here.

**Comment by 800118** [comment]
Sorry, I don't know how to edit this topic. Let me explain the detailed process of the problem. The operating system is Centos 8.5.2111, and the xrdp version used is 0.10.0-4.el8, which is an automatically updated version of the system. Previously, the xrdp version used was xrdp-0.9.25-2.el8. When logging into Centos 8 using a remote desktop on a Windows system, version 0.9.25 works fine. After updating version 0.10.0-4, the login interface changes to the image shown above, with an abnormal interface. Additionally, the username selection box cannot be selected with the mouse and content cannot be entered. That's all
PS. The content description is automatically translated using translation software, which may be difficult to understand. Please understand.

**Comment by matt335672** [comment]
Thanks - I understand I think.

It's likely this is caused by some customisations in your existing `/etc/xrdp/xrdp.ini` file.

**Take a copy of your existing `/etc/xrdp/xrdp.ini`** Then try the following:-
Try the following:-
1) `sudo dnf remove xrdp`
2) `sudo rm -f /etc/xrdp/xrdp.ini`
3) `sudo install xrdp`
4) `sudo systemctl restart xrdp`

Please report back.

**Comment by 800118** [comment]
Following the operation process you mentioned, everything was fine after reinstalling 0.9.25-2.el8. After upgrading to 0.10.0-4.el8, the screen changed to a different mode, and username could be entered normally. However, after restarting the server, it returned to the original point as shown in the above picture.
![2024-06-25_003539](https://github.com/neutrinolabs/xrdp/assets/84504466/88c678e8-d9fe-4dcd-8bb9-2d3733f926e7)
![2024-06-25_003719](https://github.com/neutrinolabs/xrdp/assets/84504466/d2acc1de-2b3b-4427-b398-26b7f185db28)
![2024-06-25_003749](https://github.com/neutrinolabs/xrdp/assets/84504466/fb2d44d9-0956-416a-8584-a636d30117b3)
![2024-06-25_003839](https://github.com/neutrinolabs/xrdp/assets/84504466/28434bd2-52bc-48af-ba38-92d65014f347)
![2024-06-25_005010](https://github.com/neutrinolabs/xrdp/assets/84504466/0155270e-07ba-4700-800b-0ab0e50f3ecf)

**Comment by 800118** [comment]
I'm very sorry, the testing was not rigorous just now. I deleted the xrdp program again, then deleted all remaining files in the/etc/xrdp/directory, and installed xrdp using yum. The screen is now normal. Thank you very much for your help and also for your xrdp
![2024-06-25_005838](https://github.com/neutrinolabs/xrdp/assets/84504466/4af993e2-4caa-4c65-a6a4-2d5f7e3a9ec8)

**Comment by matt335672** [comment]
@Aerithlon - are you able to post a copy of your original xrdp.ini?

If we can reproduce this here we may be able to help others who are experiencing the same problem.

Thanks.

**Comment by metalefty** [comment]
@matt335672 I think you can reproduce it using v0.9's xrdp.ini with v0.10.

**Comment by 800118** [comment]
This issue can be replicated in the Centos8 system. The same situation exists when installing version 0.10.0 of xrdp on two servers. The solution is the same: uninstall the old version of xrdp, delete all remaining files in the etc \ xrdp folder, and reinstall the new version of xrdp.
![2024-06-25_213150](https://github.com/neutrinolabs/xrdp/assets/84504466/bbb18958-564b-4649-871a-edd2fa5cbfbc)
![2024-06-25_213345](https://github.com/neutrinolabs/xrdp/assets/84504466/886f9519-9991-4fa1-b451-2884c175a2a0)
![2024-06-25_213420](https://github.com/neutrinolabs/xrdp/assets/84504466/4f91d9e5-a479-45c2-8459-117c2ae30fad)
![2024-06-25_213439](https://github.com/neutrinolabs/xrdp/assets/84504466/359f7762-7f8d-47b0-8737-a5f2e588e94d)
![2024-06-25_213630](https://github.com/neutrinolabs/xrdp/assets/84504466/7cb0712a-2174-4c6b-a2e7-88188a26ef86)
![2024-06-25_214350](https://github.com/neutrinolabs/xrdp/assets/84504466/1c97845e-c8c7-494f-a4b8-bd047b63ee3e)

**Comment by 800118** [comment]
I have packaged the remaining files after uninstalling the old version of xrdp.ini, among which xrdp.ini was backed up before uninstallation, and this file will be deleted after uninstallation.
[xrdpini.zip](https://github.com/user-attachments/files/15972798/xrdpini.zip)

**Comment by matt335672** [comment]
Thanks both.

I've managed to reproduce this with @Aerithlon's xrdp.ini on a fresh install of xrdp on Alma 9.

The recommendation for an upgrade from v0.9.x to v0.10.x will be to replace `xrdp.ini` with the new file wherever possible. However, this may not be possible for many administrators or administrators may wish to defer the changes until they've fully revised the file

If you're in that situation, the MINIMUM changes to the old file style to scale the logo are to find the part of the `[Globals]` section related to the logo and to add the following settings:-

```
+ls_logo_transform=scale
+ls_logo_width=240
+ls_logo_height=140
```

That results in the following screen, which is a bit off but at least works OK (although I've only tested this without scaling):-

![image](https://github.com/neutrinolabs/xrdp/assets/30179339/233cf31f-1033-44ea-a7e3-f52c3e5e42c4)

The other settings which have changed should work fine with v0.10.x. Anyone can report here if that's not the case.

@Aerithlon - I'm going to change the title of this issue report to make it easier to find on an Internet search.
